packages/dds/tree/src/test/shared-tree/fuzz/fuzzEditGenerators.ts

```diff
@@ -248,7 +287,7 @@ function getRandomPlace(tree: ISharedTree, random: IRandom): UpPath {
 	const firstField = cursor.firstField();
 	if (!firstField) {
 		cursor.free();
-		return { parent: parentPath, parentField: testerKey, parentIndex: 0 };
+		return { parent: parentPath, parentField: random.pick(fieldKeys), parentIndex: 0 };
```

Comment:
Sequence field is the only field where a place can be edited, so you can just have
```suggestion
		return { parent: parentPath, parentField: sequenceField, parentIndex: 0 };
```

Reply:
changed to hardcoded field `sequenceField`